Marlin/pins_MKS_BASE.h

```diff
@@ -26,5 +26,8 @@
 
 #include "pins_RAMPS_14_EFB.h"
 
+#undef BOARD_NAME
+#define BOARD_NAME "MKS BASE 1.0"
```

Comment:
Instead of using this approach, where we `#undef` the board name to override the name set by `pins_RAMPS_14.h`, set the `BOARD_NAME` above the `#include` line. Then in the pins files that are included by others, use, for example…
```cpp
#ifndef BOARD_NAME
  #define BOARD_NAME "RAMPS 1.3"
#endif
```